### PR TITLE
Make arrow red when previous command returned non zero exit status

### PR DESCRIPTION
**Pull request by mskelton**
This updates the prompt to set the arrow color to red if the previous command returns a non zero exit status.  See the screenshot below for how this looks:

Screen Shot 2021-03-05 at 10 24 45 AM

### CODE REVIEW

**Comment by mskelton** [comment]
@bobthecow If you have some time to give this simple PR a lookover, that would be great!

File: functions/fish_prompt.fish [hunk 1]
```diff
@@ -15,13 +15,15 @@ function _drush_alias_name
...
+  set -l code $status
+
   set -l cyan (set_color -o cyan)
   set -l yellow (set_color -o yellow)
   set -l red (set_color -o red)
   set -l blue (set_color -o blue)
   set -l normal (set_color normal)
 
-  set -l arrow "$blue➜ "
+  set -l arrow (test $code = 0; and echo "$blue➜ "; or echo "$red➜ ")
```
Comment (scorphus) [inline functions/fish_prompt.fish]: Thanks for contribuitng!

That's a nice improvement. As we're at it, please consider my suggestion below.

```suggestion
  test $code = 0; and set -l arrow "$blue➜ "; or set -l arrow "$red➜ "
```

edit: Conditionally *setting* the variable is considerably more *efficient* than conditionally *echoing* its content:

```
➜ ~ set -l code 0                                                           
➜ ~ time for i in (seq 1000)                                                
        set -l arrow (test $code = 0; and echo "$blue➜ "; or echo "$red➜ ") 
    end                                                                     
                                                                            
________________________________________________________                    
Executed in  258.09 millis    fish           external                       
   usr time  155.05 millis  154.18 millis    0.87 millis                    
   sys time  170.40 millis  168.66 millis    1.74 millis                    
                                                                            
➜ ~ time for i in (seq 1000)                                                
        test $code = 0; and set -l arrow "$blue➜ "; or set -l arrow "$red➜ "
    end                                                                     
                                                                            
________________________________________________________                    
Executed in   31.33 millis    fish           external                       
   usr time   29.45 millis   29.45 millis    0.00 millis                    
   sys time    2.69 millis    0.63 millis    2.06 millis                    
                                                                            
➜ ~                                                                         
```

**Review by scorphus — CHANGES_REQUESTED** [review]

File: functions/fish_prompt.fish (hunk 1 above)
Comment (mskelton) [reply functions/fish_prompt.fish]: Neat!  TIL

**Review by scorphus — APPROVED** [review]

**Comment by scorphus** [comment]
Thank you, @mskelton!

**Pull request merged**